lib/hets/prove/prove_evaluation_helper.rb

```diff
@@ -22,7 +22,7 @@ def fill_proof_attempt_instance(proof_attempt, proof_info)
         proof_attempt.proof_status = find_proof_status_with_hash(proof_info)
         proof_attempt.prover = find_or_create_prover_with_hash(proof_info)
         proof_attempt.prover_output = proof_info[:prover_output]
-        proof_attempt.time_taken = proof_info[:time_taken]
+        proof_attempt.time_taken = time_taken_from_hash(proof_info)
         proof_attempt.tactic_script = tactic_script_from_hash(proof_info)
```

Comment:
Why did you decide to send the hash to the method? I'm just interested in your reasoning.